src/audits/TableHasAppropriateHeaders.js

```diff
@@ -73,6 +73,34 @@ goog.require('axs.constants.Severity');
         }
         return false;
     }
+	
+    /**
+     * Checks whether a table is a layout table.
+     *
+     * @returns {boolean} Table is a layout table
+     */
+    function isLayoutTable(element) {
+        if (element.childElementCount == 0) {
+            return true;
+        }
+
+        if (element.getAttribute('role') == 'presentation') {
+            var tableChildren = element.querySelectorAll('*')
+
+            for (var i = 0; i < tableChildren.length; i++) {
+                if (tableChildren[i].tagName == 'TR' || tableChildren[i].tagName == 'TD') {
```

Comment:
Can you add a comment explaining this?

Also, I'd write it inverting the logic, i.e.:
```js
if (tableChildren[i].tagName != 'TR' && tableChildren[i].tagName != 'TD')
   return false;
```